handler.go

```diff
@@ -87,6 +87,13 @@ func (ws *wasmServer) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		if _, err := w.Write(ws.wasmExecJS); err != nil {
 			ws.logger.Println("unable to write wasm_exec.")
 		}
+	default:
+
+		if dir, err := os.Getwd(); err == nil {
```

Comment:
I'm okay with it provided we gate this with a config param. I need to review https://github.com/agnivade/wasmbrowsertest/pull/34 which reworks the flag parsing logic to pass new config params easily. We can add this after that is merged.

In the meanwhile, would you mind updating the PR with the value of the os.Getwd() stored as part of the struct in `NewWASMServer`?

Thanks.